client/components/lessons/LessonNavigation/LessonNavigation.tsx

```diff
@@ -0,0 +1,48 @@
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+import HorizontalSeparator from '~/components/HorizontalSeparator';
+import { getAdjacentLessons } from '~/services/DataModel';
+import styles from './LessonNavigation.module.scss';
+
+export default function LessonNavigation(
+  { lessonId }: PropsWithChildren<{ lessonId: string }>,
+) {
+  const [adjacentLessons, setAdjacentLessons] = useState([]);
```

Comment:
Why do we need state here? Doesn't seem like we're changing it.

Reply:
Removed. It was a leftover from the first iteration of the component